about.html

```diff
@@ -0,0 +1,119 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Polyglot</title>
+    <link rel="icon" href="IMG/logo.jpg" type="image/x-icon">
+    <link
+        href="https://fonts.googleapis.com/css2?family=Lato:ital,wght@0,100;0,300;0,400;0,700;0,900;1,100;1,300;1,400;1,700;1,900&display=swap"
+        rel="stylesheet" />
+    <link rel="stylesheet" href="about.css">
+    <script src="https://kit.fontawesome.com/ecb44d8fff.js" crossorigin="anonymous"></script>
+</head>
+
+<body>
```

Comment:
# Please **Mobile size** has a white background and same color. 
- See your design: pay attention to the color
- 
![Capture](https://user-images.githubusercontent.com/63972670/196747491-c07ec447-c499-4514-b2b4-b8300e19208b.JPG)
- See original design:
- 
![Capture](https://user-images.githubusercontent.com/63972670/196747750-9abae180-0581-4db4-90ee-4f27f34275fa.JPG)